alpine/resin

```diff
@@ -0,0 +1,66 @@
+#!/sbin/openrc-run
+
+description="Resin.io User Application"
+log_path=/var/log
+log_file=resinapp.log
+pid_file=/var/run/resinapp.pid
+waiting_time=1000
+
+# Send SIGTERM to a process tree.
+recursive_term()
+{
+	pids=$(pgrep -P $1 | tr '\n' ' ')
+	if [ -z "$pids" ]; then
+		kill -TERM $1
+	else
+		for pid in $pids; do
+			recursive_term $pid
+		done
+	fi
+}
+
+start_pre() {
+	# Start Udev manually
+	rc-service udev start
```

Comment:
@petrosagg, I don't think this will harm our image since it only spawns the udev daemon. You can read udev init script here: http://pastebin.com/CHBK1yzp